### ISSUE TITLE
Trained models

### ISSUE THREAD

**Issue by manuelpenag** [JPLeoRX/detectron2-publaynet#2]
Hi ,
Thank you for the work that you have done with this, it looks amazing. I have a question, are the trained models still available? I got a 503 error message when I tried to download them
@JPLeoRX

**Comment by JPLeoRX** [comment]
Hello, yes the models are still available, but unfortunately Keybase has a new weird bug with their cloud system. I can still access the models through mobile & desktop apps, but seems like other users are unable to do so. I've received a couple more complaints on this subject and I decided to add a [Google Drive backup](https://drive.google.com/drive/folders/11BeTAb8BlS9DiEb_ndoAh1fKyq16i6FC?usp=sharing) just to make sure these models stay accessible. Readme and download links were updated accordingly. Let me know if you have any further issues.
